aws/rust-runtime/aws-sigv4/aws-sig-v4a-test-suite/get-header-key-duplicate/context.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
can you add a readme to the partent directory discussing where these tests come from, their format, etc.